harmony/version.py

```diff
@@ -0,0 +1,21 @@
+import ast
+import re
+import os
+
+
+def get_version():
```

Comment:
I don't understand why this is necessary vs just doing `harmony.__version__`.  If it's a dependency thing, we could move `__version__` into this file and reference it from `__init__.py`